src/components/bulletin-components/BulletinList.tsx

```diff
@@ -11,18 +13,24 @@ export default class BulletinList extends React.Component<BulletinListProps> {
     const { list } = this.props;
 
     if (list.length > 0) {
+      const organizedList = list.sort((a, b) => {
+        const aDate = new Date(a.createdAt);
+        const bDate = new Date(b.createdAt);
+        return aDate > bDate ? -1 : aDate < bDate ? 1 : 0;
```

Comment:
What about like this? 
![image](https://user-images.githubusercontent.com/12229968/63962188-14d8b980-ca9b-11e9-89da-0fbcfca16874.png)

Would it be bit more easier to understand? :)